python/ray/autoscaler/ray-schema.json

```diff
@@ -337,8 +337,12 @@
                        "min_workers": {"type": "integer"},
                        "max_workers": {"type": "integer"},
                        "resources": {
-                           "type": "object",
-                           ".*": {"type": "number"}
+						   "patternProperties": {
+							   ".*":{
+								   "type":"integer",
+								   "minimum": 0
+							   }
+						   }
```

Comment:
Why this change is necessary? Not sure if someone can use here fractions (maybe? CC @wuisawesome).
Can we also ident it a little bit to the left?

Reply:
Fractional resources not allowed -- there's a thread about this on Anyscale Slack #autoscaler channel.
I had assumed that they were allowed, put a fraction in an example config, and then tracked down the internal exception that says you can't use fractions.